### PR TITLE
drake hpc tech note

### PR DESCRIPTION
**Pull request by wlandau**
At the suggestion of @stefaniebutland, this PR adds a technical note on the recent improvements to [`drake`](https://github.com/ropensci/drake)'s high-performance computing capabilities.

Could we shoot for May 18 as a release date? If not, I totally understand. The unconf is coming up very soon, after all, and there may be several other tech notes lined up.

@krlmlr, near the end of the post I try to acknowledge your generous help and guidance leading up to the improvements. Would you take a look and let me know if it is okay?

### CODE REVIEW

**Comment by stefaniebutland** [comment]
I've asked for someone to check on the failing checks

File: content/technotes/2018-05-18-drake-hpc.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+The `parallelism` argument of `make()` controls not only where to deploy the workers, but also how to schedule them.
+
+|                       | Deploy: local | Deploy: remote |
+| --------------------- |:-------------:| -----:|
+| <b>Schedule: persistent</b> | "mclapply", "parLapply" | "future_lapply" |
+| <b>Schedule: transient</b>  | | "future", "Makefile" |
+| <b>Schedule: staged</b>     | "mclapply_staged", "parLapply_staged" | |
+
+In persistent scheduling, `make(jobs = 2)` deploys three processes: two workers and one master. Whenever a worker is idle, the master assigns it the next target whose dependencies are fully ready. The workers keep running until no more targets remain.
```
[the remark this answers is not in the document]
Comment (wlandau) [reply content/technotes/2018-05-18-drake-hpc.md]: Update: I implemented both your comments, and I think they improved flow and readability. Thanks!

**Comment by stefaniebutland** [comment]
Ready to publish @wlandau ?

**Comment by wlandau** [comment]
Yes, thank you!

**Comment by krlmlr** [comment]
Thanks to rOpenSci for supporting my work!

**Pull request merged**